src/DependencyInjection/Configuration.php

```diff
@@ -5,27 +5,69 @@
 namespace Neusta\ConverterBundle\DependencyInjection;
 
 use Neusta\ConverterBundle\Converter\GenericConverter;
+use Neusta\ConverterBundle\DependencyInjection\Converter\ConverterFactory;
 use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
 use Symfony\Component\Config\Definition\ConfigurationInterface;
 
 final class Configuration implements ConfigurationInterface
 {
+    /** @var list<ConverterFactory> */
+    private array $converterFactories;
+
+    /**
+     * @param list<ConverterFactory> $converterFactories
+     */
+    public function __construct(array $converterFactories)
+    {
+        $this->converterFactories = $converterFactories;
+    }
```

Comment:
Müsste ich in meinem PR auf diese Weise nun auf vorsorglich PopulatorFactories hinzufügen?!

Reply:
In which one?